CHANGELOG.md

```diff
@@ -1,3 +1,46 @@
+# v1.19.2 (2024-02-26)
+
+## OS Changes
+* Update third party packages ([#3789])
+* Update kernel to 5.10.209, 5.15.148, 6.1.77 ([#3767], [#3797])
```

Comment:
I think we supplanted #3767 for #3797 so I don't think we need to call it out since there isn't a stacking change right?